clippy_lints/src/derive.rs

```diff
@@ -442,13 +442,25 @@ impl<'tcx> Visitor<'tcx> for UnsafeVisitor<'_, 'tcx> {
     }
 }
 
+fn has_non_exhaustive_attr(cx: &LateContext<'_>, adt: AdtDef<'_>) -> bool {
```

Comment:
This function should probably be in `clippy_utils`, it looks like there may be other lints in the future where it might be handy.

Reply:
Moving it then. :+1: